app/views/users/confirm_urs_association.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="urs-assocation">
+  <div class="row row-content">
+    <div class="content">
+      <section>
+        <h2>Confirm URS and Launchpad account association</h2>
+
+        <fieldset>
+          <p>Dear <%= "#{@profile.fetch('first_name', '')} #{@profile.fetch('last_name', '')}," %> </p>
```

Comment:
Double check with Alicia and Thomas that this is the language we want to use in these two paragraphs. Just thinking that in the past any time we have more than a simple line of text we made sure that it was approved by Kathy first

Reply:
@lhlyles has a ticket to verify the language with Alicia and update with any text changes, which I think is a better process than what we had previously

Reply:
yea, this text does not have to be perfect, we can that updated latter.